tests/assertions/ocp4/rhcos4-high-4.16.yml

```diff
@@ -574,7 +574,7 @@ rule_results:
   e2e-high-master-partition-for-var-log:
     default_result: NOT-APPLICABLE
```

Comment:
This might also need to be adjusted based on additional CI runs.

https://prow.ci.openshift.org/view/gs/test-platform-results/logs/periodic-ci-ComplianceAsCode-content-master-4.17-e2e-aws-rhcos4-high-weekly/1897429479077187584